client/src/components/DailyPlanner.tsx

```diff
@@ -226,13 +223,13 @@ function DailyPlanner (props: any, ref: any) {
 
     const activityId = ev.dataTransfer.getData('text');
     // TODO: adapt for Backend when the time comes.
```

Comment:
Non-serious review: this sounds so dramatic :))

Reply:
Yeah, things are so uncertain these days :laughing:.